test/fixtures/linter/rules/NoDeprecatedApi/Control_ManagedObject.js

```diff
@@ -0,0 +1,67 @@
+sap.ui.define(["sap/ui/core/Control", "sap/m/library", "mycustom/lib/FancyMultiPage"], 
+	function (Control, library, FancyMultiPage) {
+	var DateTimeInputType = library.DateTimeInputType;
+	var FrameType = library.FrameType;
+	return Control.extend("sap.fancy.Text", {
+		metadata: {
+			// sap.ui.commons.FormattedTextViewControl: deprecated
+			interfaces: ["sap.ui.commons.FormattedTextViewControl"],
+			library: "sap.fancy",
+			properties: {
+				text: {type: "string", group: "Data", defaultValue: "", bindable: "bindable"},
+				textShort: "sap.f.AvatarShape",
+				// sap.m.DateTimeInputType: deprecated, DateTimeInputType.DateTime: deprecated
+				textDirection: {type: "sap.m.DateTimeInputType", group: "Appearance", defaultValue: DateTimeInputType.DateTime},
+				// sap.m.DateTimeInputType: deprecated, "Date": deprecated
+				textDirectionB: {type: "sap.m.DateTimeInputType", group: "Appearance", defaultValue: "Date"},
+				// sap.m.FrameType: NOT deprecated, sap.m.FrameType.TwoThirds: deprecated
+				textAlign: {type: "sap.m.FrameType", group: "Appearance", defaultValue: FrameType.TwoThirds},
+				// sap.m.FrameType: NOT deprecated, "TwoThirds": deprecated
+				textAlignB: {type: "sap.m.FrameType", group: "Appearance", defaultValue: "TwoThirds"},
+			},
+			aggregations: {
+				// ap.m.P13nColumnsItem: deprecated
+				myagg: {type: " sap.m.P13nColumnsItem[]", multiple: false, visibility: "hiddenDeprecated"},
```

Comment:
I don't think that this definition is correct. The aggregation is not even typed as "multiple".
The cases I could find were mainly using the array notation within event parameters, see https://github.com/SAP/openui5/blob/82b2165a325f8ff2f5388195e176929f39ded271/src/sap.m/src/sap/m/P13nColumnsPanel.js#L151

Reply:
Thanks!